Dockerfile

```diff
@@ -43,18 +45,23 @@ RUN apk update --no-cache && \
 RUN mix local.rebar --force && \
     mix local.hex --force
 
-# Copy dependency config first
+# Install Hex dependencies
 COPY mix.* ./
-RUN mix deps.get --only ${MIX_ENV}
-
-# Copy codebase and compile
-COPY . .
+RUN mix deps.get --only ${MIX_ENV} && \
+    mix deps.compile
+
+# Compile codebase
+COPY config config
+COPY lib lib
+COPY priv priv
+COPY rel rel
```

Comment:
Do we need the `rel` directory to run `mix compile` or could we only copy it just before running `mix release`?

Reply:
Excellent question; I’ll give it a try!

My guess is your right, we could move it down before `mix release`…

Reply:
I does work 🥇